tests/installation/releasenotes.pm

```diff
@@ -27,7 +27,12 @@ sub run() {
         send_key "alt-e";    # open release notes window
     }
     else {
-        send_key "alt-l";    # open release notes window
+        if (check_var('VIDEOMODE', 'text')) {
+            send_key "alt-l";    # open release notes window
+        }
+        else {
+            assert_and_click('release-notes-button');
+        }
```

Comment:
I think I know the answer, but I guess that's the reason why was releasenotes test removed from released products